v0/destinations/hs/transform.js

```diff
@@ -76,7 +76,6 @@ async function getTransformedJSON(message, mappingJson, destination) {
 
   if (traits) {
     const traitsKeys = Object.keys(traits);
-    const propertyMap = await getProperties(destination);
```

Comment:
I would still keep the `destination` argument and calculate `propertyMap` if the incoming value is not defined.

Reply:
added the check